feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -133,11 +130,11 @@ func TestHardwarecards(t *testing.T) {
 			rrValidation:          false,
 			operStatus:            activeStatus,
 			parent:                "",
-			pType:                 componentType["chassis"],
+			pType:                 chassisType.String(),
```

Comment:
Could you change the parent and pType fields to be of type oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT?

Reply:
pType still needs to be changed to be of type oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT

Reply:
Changed pType's type. Could not change parent's type to oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT since the field is being set to empty for certain components in order to skip the check for that leaf; and if it is of type oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT, I am unable to set it to an empty value.

Reply:
Use oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT for the parent type and use oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_UNSET as the empty value. (That's the zero value it will default to if you just don't specify the field).